stackoverflow-dark.user.css

```diff
@@ -1409,21 +1410,21 @@ regexp("^https?:\/\/((?!area51)\w+\.)*?stackexchange\.com.*") {
   /* prevent animation from breaking while loading  issue #187 */
   @keyframes blinking {
     from {
-      background: rgba(0, 40, 16, .8);
+      background: rgba(/*[[accepted-color-rgb]]*/, .8);
     }
     to {
-      background: rgba(0, 40, 16, .5);
+      background: rgba(/*[[accepted-color-rgb]]*/, .5);
     }
   }
   #answers .answer.accepted-answer[style*="background-color"] {
     animation-name: blinking !important;
-    background: #001c0b !important;
-    border: 1px solid #13552f !important;
+    background: rgb(/*[[accepted-color-rgb]]*/) !important;
```

Comment:
```suggestion
    background: inherit !important;
```

Reply:
for some reason I cant commit my suggestions @theslimshaney have you checked this box? 

![Capture](https://user-images.githubusercontent.com/31389848/73256926-1f6eba80-41c3-11ea-9c24-d3ab3ed08c21.PNG)


Its at bottom of PR sidebar. If you haven't please do check it, it makes this process more efficient if I can commit the fixes directly to this =)

Reply:
@the-j0k3r I do have it checked, I don't know why it's not working :( I tried inviting you as collaborator as well, maybe that will help.

I will try to get to these changes today, otherwise I won't get time until the weekend. Feel free to make the changes separately if you'd like. I tried out the changes last night and ran into some issues while testing it so I didn't want to push just yet.

Reply:
> I do have it checked, I don't know why it's not working :( I tried inviting you as collaborator as well, maybe that will help.

yea it shouldn't be necessary to be a collaborator in your fork, Its a permissions issue maybe due to the settings on this repo.

@silverwind can you please check that Im allowed to commit suggestions, maybe better write perms?

Reply:
@theslimshaney please check youre setup as https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests indicates.

I still cant push the suggestions and its not a permissions issue on the repo setup.

Reply:
![image](https://user-images.githubusercontent.com/33791263/74211612-879eb100-4c45-11ea-8393-ad716647b99d.png)

@the-j0k3r Already have it checked, any other ideas?

Reply:
Try to accept and commit the suggestions (by clicking commit the suggestion buttons) then so we can move this fwd =)